src/main/kotlin/omok/domain/Position.kt

```diff
@@ -17,8 +17,10 @@ data class Position(val horizontalAxis: HorizontalAxis, val verticalAxis: Int) {
     companion object {
         private const val MIN_VERTICAL_AXIS = 1
         private const val MAX_VERTICAL_AXIS = 15
-        private const val HORIZONTAL_AXIS_ERROR_MSG = "가로축은 A와 O 사이 입니다."
-        private const val VERTICAL_AXIS_ERROR_MSG = "세로축은 1과 15 사이 입니다."
+        private const val MIN_HORIZONTAL_AXIS = "A"
+        private const val MAX_HORIZONTAL_AXIS = "O"
+        private const val HORIZONTAL_AXIS_ERROR_MSG = "가로축은 %s와 %s 사이입니다."
```

Comment:
```suggestion
        private const val HORIZONTAL_AXIS_ERROR_MSG = "가로축은 $MIN_HORIZONTAL_AXIS와 $MAX_HORIZONTAL_AXIS 사이입니다."
```
format을 사용해주셔도 좋지만, 이렇게 작성해주셔도 됩니다. 😄

Reply:
넵! 추천해주신 방식이 더 간단한 것 같아 추천 방법으로 수정했습니다!